src/main/java/org/jbake/app/Asset.java

```diff
@@ -81,6 +81,35 @@ public boolean accept(File file) {
             }
         }
     }
+    
+    public void copyAssetsFromContent(File path){
+    	copyNonContentFiles(path, destination);
+    }
+    
+    private void copyNonContentFiles(File sourceFolder, File targetFolder) {
```

Comment:
This is a duplication of the copy method in Line [55](https://github.com/jbake-org/jbake/pull/390/files#diff-c0fef1d7b85aafda758d7f9855f5d31eL55). The only difference is the FileFilter applied to the sourceFolder.

So it would be nice to refactor it to something like:

```
	/**
	 * Copy all files from supplied path.
	 *
	 * @param path	The starting path
	 */
	public void copy(File path) {

	    FileFilter filer = new FileFilter() {
            @Override
            public boolean accept(File file) {
                return (!ignoreHidden || !file.isHidden()) && (file.isFile() || FileUtil.directoryOnlyIfNotIgnored(file));
            }
        };
	    copy(path, destination, filer);
    }

    public void copyAssetsFromContent(File path){

        FileFilter filer = FileUtil.getNotContentFileFilter();
        copy(path, destination, filer);
    }

    private void copy(File sourceFolder, File targetFolder, FileFilter filter) {
        final File[] assets = sourceFolder.listFiles(filter);
        if (assets != null) {
            Arrays.sort(assets);
            for (File asset : assets) {
                final File target = new File(targetFolder, asset.getName());
                if (asset.isFile()) {
                    final StringBuilder sb = new StringBuilder();
                    sb.append("Copying [").append(asset.getPath()).append("]... ");
                    try {
                        FileUtils.copyFile(asset, target);
                        sb.append("done!");
                        LOGGER.info(sb.toString());
                    } catch (IOException e) {
                        sb.append("failed!");
                        LOGGER.error(sb.toString(), e);
                        errors.add(e);
                    }
                } else if (asset.isDirectory()) {
                    copy(asset, target, filter);
                }
            }
        }
    }
```